.github/workflows/python-poetry-publish.yaml

```
@@ -0,0 +1,53 @@
+name: Reusable workflow for publishing to PyPI
+
+on:
+  workflow_call:
+    inputs:
+      publish-to-test:
+        description: "If set to false, the packages are published to PyPI. (Default is true)"
+        required: false
+        default: "true"
+        type: string
+      python-version:
+        description: "The Python version for the Poetry virtual environment. (Default is 3.10)"
+        required: false
+        default: "3.10"
+        type: string
+      poetry-version:
+        description: "The Poetry version to be installed. (Default is 1.5.1)"
+        required: false
+        default: "1.5.1"
+        type: string
+      working-directory:
+        description: "The working directory of your Python packages. (Default is root directory)"
+        required: false
+        default: "./"
+        type: string
+      ref:
+        description: "The ref name to checkout the repository."
+        required: false
+        default: ${{ github.event.repository.default_branch }}
+        type: string
+    secrets:
+      pypi-token:
+        description: "PyPI token"
+        required: true
+
+jobs:
+  publish:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.ref }}
+
+      - name: Publish to PyPI
+        id: publish
+        uses: bakdata/ci-templates/actions/python-poetry-publish@v1.21.2
```

this will be a minor bump
```suggestion
        uses: bakdata/ci-templates/actions/python-poetry-publish@v1.22.0
```